src/makielayout/types.jl

```diff
@@ -126,7 +126,7 @@ mutable struct LineAxis
     elements::Dict{Symbol, Any}
     tickpositions::Observable{Vector{Point2f}}
     tickvalues::Observable{Vector{Float32}}
-    ticklabels::Observable{Vector{String}}
+    ticklabels::Observable{Vector{AbstractString}}
```

Comment:
This change is not needed for the fix.
I thought I include it because otherwise `LaTeXStrings` decay here too.

Reply:
I guess that's fair. Just goes to show how difficult it is to type these fields statically